pkg/serviceaccounts/oauthclient/oauthclientregistry_test.go

```diff
@@ -142,7 +142,7 @@ func TestGetClient(t *testing.T) {
 						UID:       types.UID("any"),
 						Annotations: map[string]string{
 							OAuthRedirectModelAnnotationURIPrefix + "one":     "http://anywhere",
-							OAuthRedirectModelAnnotationReferencePrefix + "1": buildRedirectObjectReferenceString(routeKind, "route1", ""),
+							OAuthRedirectModelAnnotationReferencePrefix + "1": buildRedirectObjectReferenceString(routeKind, "route1", "route.openshift.io"),
```

Comment:
Opened https://github.com/mfojtik/origin/pull/27 to fix

cc @liggitt @gabemontero